clayui.com/content/docs/components/css-nav.md

```diff
@@ -382,6 +384,70 @@ Add `nav-unstyled` to your nav to remove spacing around `nav-link` and `nav-btn`
 </ul>
 ```
 
+### Nav Pills
```

Comment:
Going off of chema's recommendation, I think we can just remove any reference to nav-pills since its really just a bootstrap thing and not necessarily a Lexicon thing. https://github.com/liferay/clay/issues/3205#issuecomment-626547433